dipy/viz/fvtk.py

```diff
@@ -79,6 +79,9 @@
     msg = "Python VTK is not installed"
     warn(msg)
 
+    ren, have_ren, _ = optional_package('dipy.viz.window.ren',
```

Comment:
@garyfallidis - I don't understand this. Is there any reason line 80 doesn't raise an error? If have_vtk is False, we're bound to hit an error sooner or later, no?